packages/amplication-client/src/Project/ArchitectureConsole/ApplyChangesNextSteps.tsx

```diff
@@ -92,12 +51,13 @@ export const ApplyChangesNextSteps = ({
         <Text textStyle={EnumTextStyle.H3}>What should we do next?</Text>
       </FlexItem>
       <div className={`${className}__box_container`}>
-        <JumboButton
-          onClick={handleGenerateCodeClicked}
-          text="Generate the code for my new architecture"
-          icon="pending_changes"
-          circleColor={EnumTextColor.ThemeTurquoise}
-        ></JumboButton>
+        <Commit
+          projectId={currentProject.id}
+          noChanges
+          showCommitMessage={false}
+          commitMessage="Generated code with my new architecture"
```

Comment:
did @PazYanoverr provided this message?

Reply:
No...
@morhag90 - Is it the commit comment when applying the architecture changes?

Reply:
@PazYanoverr  It is the commit message that will be shown on the commit page: 
<img width="266" alt="image" src="https://github.com/amplication/amplication/assets/97830649/66267355-3fa1-486f-a510-2eb226cb3a35">

Reply:
@morhag90 - Can we please update it to "Architecture redesign"?